server/marketplace/marketplace.go

```diff
@@ -0,0 +1,70 @@
+package marketplace
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+
+	"time"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+const (
+	// marketplaceVersion is the URL version of Marketplace API.
+	marketplaceVersion = "/api/v1"
+
+	// listPluginsEndpoint is an endpoint to get plugin list.
+	listPluginsEndpoint = "/plugins"
```

Comment:
Do we have any specific advantage to have 2 different constants?

Reply:
I agree, I think we don't need it. I thought it might make sense to have this separated when API version is updated but `/plugins` endpoint remains the same. But this is not guaranteed and we might use different versions of API for different endpoints anyway.

I'll be merging these two into one.

Reply:
done by 556f760477a45a208ac06624deb8d26933b9c13d